compose-builder/docker-compose-base-zero-trust.yml

```
@@ -0,0 +1,91 @@
+name: edgex
+services:
+  app-rules-engine:
+    environment:
+      SERVICE_HOST: app-rules-engine.edgex.ziti
+      SERVICE_PORT: 80
+  core-command:
+    environment:
```

Do this for all the services.
```suggestion
    ports: !reset null
    environment:
```